print('Hello World')
try:
  print(x)
except:
  print("An exception occurred")
